testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oauth/AccessTokenTest.java

```diff
@@ -1088,6 +1090,50 @@ public void accessTokenRequest_RealmRS256_ClientES512_EffectiveES512() throws Ex
         }
     }
 
+    @Test
+    public void clientAccessTokenLifespanOverride() {
+        ClientResource client = ApiUtil.findClientByClientId(adminClient.realm("test"), "test-app");
+        ClientRepresentation clientRep = client.toRepresentation();
+
+        RealmResource realm = adminClient.realm("test");
+        RealmRepresentation rep = realm.toRepresentation();
+
+        int sessionMax = rep.getSsoSessionMaxLifespan();
+        int accessTokenLifespan = rep.getAccessTokenLifespan();
+
+        // Make sure realm lifespan is not same as client override
+        assertNotEquals(accessTokenLifespan, 500);
+
+        try {
```

Comment:
prefer:
```java
try (AutoCloseable cl = ClientAttributeUpdater.forClient(adminClient, "test", ""test-app")
   .setAttribute(OIDCConfigAttributes.ACCESS_TOKEN_LIFESPAN, "500")
   .update()) {
```
to lines 1095-1107 and 1132-1134.

Reply:
I don't ;)